Libraries/TestReport.psm1

```diff
@@ -252,6 +252,19 @@ Class JUnitReportGenerator
 	}
 }
 
+Class TestResult
```

Comment:
Can you make the class name TestResults? Class and method are having the same name.

Reply:
Are you referring to the method in this class? If yes, it's the constructor and it must be the same name as the class